sanic/worker.py

```diff
@@ -115,7 +115,9 @@ def run(self):
             coros = []
             for conn in self.connections:
                 if hasattr(conn, "websocket") and conn.websocket:
-                    coros.append(conn.websocket.close_connection(force=True))
+                    coros.append(
+                        conn.websocket.close_connection(after_handshake=False)
```

Comment:
Shouldn't this only be false if the handshake fails?

https://github.com/aaugustin/websockets/blob/4.0/websockets/protocol.py#L704-L705

Or am I reading that wrong?

Reply:
This line intends to `force` close connection after graceful timeout.

Reply:
If `after_handshake=True`, it'll still wait for data transfer phase or so. then we can't shutdown server even after timeout. @r0fls